src/sysupdate/sysupdate-update-set-flags.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+typedef enum UpdateSetFlags {
+        UPDATE_NEWEST    = 1 << 0,
+        UPDATE_AVAILABLE = 1 << 1,
+        UPDATE_INSTALLED = 1 << 2,
+        UPDATE_OBSOLETE  = 1 << 3,
+        UPDATE_PROTECTED = 1 << 4,
+} UpdateSetFlags;
+
+const char *update_set_flags_to_color(UpdateSetFlags flags);
+const char *update_set_flags_to_glyph(UpdateSetFlags flags);
+const char *update_set_flags_to_string(UpdateSetFlags flags);
```

Comment:
```suggestion
const char* update_set_flags_to_color(UpdateSetFlags flags);
const char* update_set_flags_to_glyph(UpdateSetFlags flags);
const char* update_set_flags_to_string(UpdateSetFlags flags);
```

according to our coding style (also at other places)

Reply:
This commit was already merged as part of https://github.com/systemd/systemd/pull/33520. I opened https://github.com/systemd/systemd/pull/33722 to fix. Please take a look.